backend/config/corpora-api.yml

```diff
@@ -548,7 +548,13 @@ paths:
                   properties:
                     id:
                       $ref: "#/components/schemas/collection_uuid"
-                    created_at:
+                    name: 
+                      type: string
+                    publisher_metadata:
+                      $ref: "#/components/schemas/publisher_metadata"
+                    published_at:
+                      type: number
+                    revised_at:
```

Comment:
why remove `created_at`?

Reply:
FE no longer uses, I assume

Reply:
I think that was a mistake in the first place, `created_at` isn't among the returned fields by this endpoint, so I just removed it.